po/ru.po

```diff
@@ -1357,44 +1389,77 @@ msgid ""
+"По нашему опыту, ученики считают полезным наличие перерывов в курсе, так как это помогает им лучше осмыслить информацию.\n"
+"\n"
+"1. Найти помещение, достаточно просторное для очного участия.\n"
+"Мы рекомендуем, чтобы в классе было 15-20 человек.\n"
+"Это достаточно немного, чтобы людям было комфортно задавать вопросы —\n"
+"это также достаточно, чтобы у одного преподавателя было время ответить на вопросы.\n"
+"\n"
+"1. В день занятия приходите в комнату немного раньше, чтобы всё подготовить.\n"
+"Мы рекомендуем показывать слайды используя `mdbook serve`, запущенного на вашем ноутбуке (см. [installation instructions][3]).\n"
+"Это обеспечивает оптимальную производительность без задержек при смене страниц.\n"
+"Использование ноутбука также позволит вам исправлять опечатки по мере их обнаружения вами или участниками курса.\n"
+"\n"
+"1. Позвольте участникам решать упражнения самостоятельно или в небольших группах.\n"
+"Обычно мы уделяем упражнениям по 30-45 минут утром и во второй половине дня (включая время на разбор решений).\n"
+"Обязательно спросите, не испытывают ли они трудности и есть ли что-нибудь, с чем вы можете помочь.\n"
+"Когда вы видите, что у нескольких человек одна и та же проблема, сообщите об этом классу и предложите решение,\n"
+"например, показав, где найти соответствующую информацию в стандартной библиотеке."
 
 #: src/running-the-course.md:43
 msgid ""
 "That is all, good luck running the course! We hope it will be as much fun for\n"
 "you as it has been for us!"
-msgstr ""
+msgstr "На этом всё, удачи в прохождении курса! Мы надеемся, что вам будет так же весело, как и нам!"
 
 #: src/running-the-course.md:46
 msgid ""
 "Please [provide feedback][1] afterwards so that we can keep improving the\n"
 "course. We would love to hear what worked well for you and what can be made\n"
 "better. Your students are also very welcome to [send us feedback][2]!"
 msgstr ""
+"Пожалуйста, [оставьте отзыв][1],\n"
+"чтобы мы могли продолжать совершенствовать курс.\n"
+"Мы хотели бы услышать, что было хорошо и что можно сделать лучше.\n"
+"Ваши студенты также могут [присылать нам свои отзывы][2]!"
 
 #: src/running-the-course/course-structure.md:1
 msgid "# Course Structure"
-msgstr ""
+msgstr "# Структура курса"
 
 #: src/running-the-course/course-structure.md:5
 msgid "The course is fast paced and covers a lot of ground:"
-msgstr ""
+msgstr "Курс проходит в быстром темпе и охватывает много вопросов:"
 
 #: src/running-the-course/course-structure.md:7
 msgid ""
 "* Day 1: Basic Rust, ownership and the borrow checker.\n"
 "* Day 2: Compound data types,  pattern matching, the standard library.\n"
 "* Day 3: Traits and generics, error handling, testing, unsafe Rust."
 msgstr ""
+"* День 1: Базовый Rust, владение и анализатор заимствований.\n"
+"* День 2: Составные типы данных, сопоставление с образцом, стандартная библиотека.\n"
```

Comment:
pattern - шаблон

Reply:
This translation taken from [this](https://github.com/rust-lang-ru/dictionary#readme) dictionary, I suggest we keep it as is.

Reply:
Ок. Но это странно. Шаблон - стандартный термин для этого, в то время как "образец" имеет очень широкий размытый спектр значений и применений и затрудняет понимание, вводя лишнее усилие на парсинг на стороне юзера.

Reply:
В rust book на удивление переводят и так и так - https://doc.rust-lang.ru/book/ch06-00-enums.html и https://doc.rust-lang.ru/book/ch18-00-patterns.html Но "pattern matching" как фраза там тоже как "сопоставление с образцом" переведена.

Reply:
почему мы ссылаемся на чей-то словарик? что делает его авторитетным?

Reply:
It's not a question of authority, these people used these dictionary to translate their books / materials. So, it would be cool to not reinvent the wheel and just use already provided translations, especially since it's a decent dictionary.

Regarding the translation, I think both these translations are correct. I will keep it as is, since don't see a reason to change that.

Reply:
It is the question of authoritativeness. Just because someone somewhere did something earlier than you. does not make them authoritative. And this dictionary is exactly an example - it is just wrong. The whole industry uses settled terms for years if nod decades, and this vocabulary has no idea about that.

Reply:
I do not want to start an argument about the correctness of this dictionary. In my opinion, this is a good dictionary that gives many useful translations.
Speaking of this translation, it will be `сопоставление с образцом`

Reply:
С шаблонами есть другая проблема - в плюсах есть шаблоны (templates) и люди могут понять как на отсылку на них, а не на паттерны. С другой стороны design patterns - это жестко и четко "Шаблоны проектирования". Может быть в скобочках указывать английские термины при переводе?

Reply:
The problem is not in useful translations in that dictionry. It is in wrong ones. By promoting wrong terms we are promoting chaos and sometime prevent the reader from understanding.